edge-api-routes/wasm-hello-world/wasm/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "wasm"
+version = "0.1.0"
+authors = ["Guillaume Bibeau-Laviolette <g.bibeaulaviolette@gmail.com>"]
```

Comment:
We shouldn't include personal emails in examples

```suggestion
```

Reply:
Thanks, I squashed this in